import-export-cli/box/resources/sample/sample_config.yaml

```diff
@@ -1,18 +1,20 @@
 config:
   export_directory: /home/wso2user/.wso2apictl/exported
   http_request_timeout: 10000
+  kubernetes_mode: false
+  token_type: JWT
 environments:
   sample-env1:
-    admin_endpoint: "https://localhost:9443/api/am/admin/v0.16"
-    api_list_endpoint: "https://localhost/publisher/apis"
-    api_manager_endpoint: "https://localhost/apim"
-    application_list_endpoint: "https://localhost:9443/api/am/admin/v0.16/applications"
-    registration_endpoint: "https://localhost/register"
-    token_endpoint: "https://localhost/token"
+    admin_endpoint: https://localhost:9443/api/am/admin/v0.16
+    api_list_endpoint: https://localhost:9443/api/am/publisher/v0.16/apis
```

Comment:
api_list_endpoint: https://localhost:9443/api/am/publisher/v1/apis

Reply:
Fixed in https://github.com/wso2/product-apim-tooling/pull/190